File: leet_codes/1470_Shuffle_the_array/shuffle.rb
nums = [1,2,3,4,4,3,2,1]
n = nums.size/2
result = []
nums.each_index do  |i| 
    if i < n 
        result <<  nums[i]
        result << nums[n+i]
    end
end

p nums
p result

File: leet_codes/1470_Shuffle_the_array/shuffle.js
const p = console.log;
let nums = [1,2,3,4,4,3,2,1];
let n = nums.length/2;
let result = []
for(let i =0; i < n; i++){
    result.push(nums[i]);
    result.push(nums[n+i]);
};
p(nums);
p(result);